### PR TITLE
Added support for Swrve (http://www.swrve.com/)

### PR DESCRIPTION
**Pull request by jhurliman**

### CODE REVIEW

File: Providers/SwrveProvider.m [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+- (id)initWithAppID:(NSString *)appID apiKey:(NSString *)apiKey {
+    [Swrve sharedInstanceWithAppID:appID.intValue apiKey:apiKey];
+    return [super init];
+}
+
+- (void)identifyUserWithID:(NSString *)userID andEmailAddress:(NSString *)email {
+    if (!userID && !email) return;
+    
+    NSMutableDictionary *traits = NSMutableDictionary.dictionary;
+    if (userID) traits[@"id"] = userID;
```
Comment (orta) [inline Providers/SwrveProvider.m]: you're already checking for userID & email's existence, may as well not use a mutable dictionary at all here

**Comment by jhurliman** [comment]
@orta I updated the PR to avoid the NSMutableDictionary while still sanely handling the cases of userID and/or email being nil.

**Comment by orta** [comment]
:+1: Cool, thanks!

**Comment by jhurliman** [comment]
Thanks for the quick review+merge. Would it be possible to get a version bump and republish the cocoapod?

**Comment by orta** [comment]
not free till next week to do that

**Pull request merged**